.github/workflows/run-metalgateway-functional-tests.yml

```diff
@@ -0,0 +1,61 @@
+# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
+
+name: Execute functional tests for MetalGateway related workflows
+
+on:
+  workflow_dispatch:
+    inputs:
+      metro:
+        description: 'metro code where you want to deploy'
+        default: 'DA'
+        required: true
+      privateIpv4SubnetSize:
+        description: 'private ipv4 subnet size'
+        default: '16'
+        required: true
+      vxlan:
+        description: 'vxlan id'
+        default: '100'
+        required: true
+      token:
```

Comment:
I think the `token` field will need to be marked as a `secrets` rather than `inputs`.
https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#example-2

Reply:
Resolved in 44f4da9ce9c3d2707530cf7aae5023b36789d7da